templates/tapestry.js

```diff
@@ -282,24 +276,14 @@ function tapestryTool(config){
         // 3. SET NODES/LINKS AND CREATE THE SVG OBJECTS
         //---------------------------------------------------
 
-        if (!isReload) {
-            svg = createSvgContainer(TAPESTRY_CONTAINER_ID);
-        }
-
-        if (!isReload && typeof tapestry.dataset.settings.defaultDepth !== "undefined") {
-            tapestryDepth = tapestry.dataset.settings.defaultDepth
-        }
-
-        addDepthToNodes(root, 0, []);
         setNodeTypes(root);
         setLinkTypes(root);
         addDepthToNodes(root, 0, []);
 
-        // add in the controls
         if (!isReload) {
-            document.getElementById(TAPESTRY_CONTAINER_ID).prepend(tapestry.getControls());
+            svg = createSvgContainer(TAPESTRY_CONTAINER_ID);
         }
-        
```

Comment:
Were these changes intentional? They don't look related to the code.

Reply:
I've re-aligned it with current master